gui/default/index.html

```diff
@@ -344,40 +344,132 @@ <h3 class="panel-title">
+                  </label>
+
+                  <div class="form-group text-right">
+                    <button type="submit" id="submit" class="btn btn-primary" ng-disabled="login.inProgress" translate>Log In</button>
+                  </div>
+
+                  <div class="row">
+                    <div class="col-md-9 login-form-messages">
+                      <p ng-if="login.errors.badLogin" class="text-danger" translate>
+                        Incorrect user name or password.
+                      </p>
+                      <p ng-if="login.errors.failed" class="text-danger" translate>
+                        Login failed, see Syncthing logs for details.
+                      </p>
+                    </div>
+                  </div>
+                </form>
+              </div>
             </div>
-            <div class="col-md-3 text-right">
-              <button type="submit" id="submit" class="btn btn-default" ng-disabled="login.inProgress" translate>Log In</button>
+          </div>
+
+          <hr class="divider" />
+
+          <div class="panel panel-default">
+            <button class="btn panel-heading" data-toggle="collapse" data-parent="#login-options" data-target="#login-option-webauthn" aria-expanded="{{isDefaultLoginOption('webauthn') ? 'true' : 'false'}}">
+              <h4 class="panel-title">
+                <div class="panel-title-text" translate>WebAuthn (Passkey)</div>
+              </h4>
+            </button>
+            <div id="login-option-webauthn" class="panel-collapse collapse {{isDefaultLoginOption('webauthn') ? 'in' : ''}}">
+              <div class="panel-body">
+                <label class="stay-logged-in">
+                  <input type="checkbox" ng-model="login.stayLoggedIn" ng-disabled="login.inProgress">&nbsp;<span translate>Stay logged in</span>
+                </label>
+
+                <div class="form-group text-right">
+                  <button type="button" class="btn btn-primary" ng-click="authenticateWebauthnFinish()" ng-disabled="login.inProgress || webauthn.errors.noCredentials || webauthn.errors.initFailed || locationDoesNotMatchWebauthnRpId()" tabindex="3">
+                    <span translate>Log in with WebAuthn</span>
+                  </button>
+                </div>
+
+                <p class="text-info" ng-if="!webauthnAvailable()" translate>
+                  Your browser does not support WebAuthn.
+                </p>
+                <div ng-if="locationDoesNotMatchWebauthnRpId()">
+                  <p class="text-info" translate>
+                    Current domain does not match WebAuthn configuration.
+                  </p>
+                  <p>
+                    <a href="{{docsURL('users/webauthn#webauthn-custom-gui-address')}}" translate>
+                      Syncthing Documentation: Customizing the GUI address
+                    </a>
```

Comment:
I'd trim white-space inside the inline tag again, reducing it to a single line.

Reply:
Done (also several other similar occurrences) in da4cd0d5c23a19869eaabcc0094d2b1bee6d7019.